src/main/java/com/spotify/docker/AbstractDockerMojo.java

```diff
@@ -246,25 +252,35 @@ protected AuthConfig authConfig() throws MojoExecutionException, SecDispatcherEx
         }
 
         return authConfigBuilder.build();
-      } else if (useConfigFile != null && useConfigFile){
-
-          final AuthConfig.Builder authConfigBuilder;
-          try {
-            if (!isNullOrEmpty(registryUrl)) {
-              authConfigBuilder = AuthConfig.fromDockerConfig(registryUrl);
-            } else {
-              authConfigBuilder = AuthConfig.fromDockerConfig();
-            }
-          } catch (IOException ex){
-            throw new MojoExecutionException(
-                      "Docker config file could not be read",
-                      ex
-            );
+      } else if (useConfigFile != null && useConfigFile) {
```

Comment:
Good point. I must have totally missed that this branch in logic even existed when making this change.